core/chains/evm/bulletprooftxmanager/common.go

```diff
@@ -34,8 +34,10 @@ func batchSendTransactions(
 
 	reqs := make([]rpc.BatchElem, len(attempts))
```

Comment:
Delete the comments in line 22-23. about the future improvement idea.
And thanks for implementing it  :)